packages/documentation/src/stories/utilities/opacity/opacity.styles.scss

```
@@ -0,0 +1,13 @@
+@use '@swisspost/design-system-styles/core' as post;
+
+.opacity-content {
```

You could style the `.opacity-*` directly :
```suggestion
[class^="opacity-"] {
```